src/components/case_manager/PersonTable.tsx

```diff
@@ -117,7 +117,7 @@ function EnhancedTableToolbar({
           width: "100%",
         }}
       >
-        <h2 className={styles.tableTitle}>{type}</h2>
+        <h2 className={styles.tableTitle}>{type} List</h2>
```

Comment:
🐧 : Let's keep it as "Staff" and "Students" for now. It matches the mockups and saying list feels redundant to me.